scripts/dev-install.py

```diff
@@ -35,9 +35,7 @@ def install_dev(no_occ_build=False):
             execute("jupyter server extension enable jupytercad_core")
 
         if py_package != "jupytercad_app":
-            execute(
-                f"jupyter labextension develop {python_package_prefix}/{py_package} --overwrite"
-            )
+            execute(f"pip install -e {python_package_prefix}/jupytercad")
```

Comment:
I'm confused about this change? We should continue installing the labextension in the dev install script

Reply:
Ahh, i just copied this from jupytergis PR, should i revert?

Reply:
I should review my own PRs 😆 I should not have removed it in the original PR

Reply:
so, the `jupyter labextension develop` command should stay, and the `execute(f"pip install -e {python_package_prefix}/jupytercad")` should stay too

Reply:
Done :)